website/src/react-native/css/react-native.css

```diff
@@ -1507,6 +1511,10 @@ div[data-twttr-id] iframe {
   .showcaseSection .inner-content {
     width: 100%;
   }
+
```

Comment:
GitHub doesn't show this in the default view, but please note that this block is within a @media query and is not in conflict with L915.